tools/clang/lib/SPIRV/CapabilityVisitor.cpp

```diff
@@ -23,6 +23,17 @@ void CapabilityVisitor::addExtension(Extension ext, llvm::StringRef target,
     spvBuilder.requireExtension(featureManager.getExtensionName(ext), loc);
 }
 
+void CapabilityVisitor::addExtensionAndCapabilitiesIfEnabled(
```

Comment:
Would you be OK with having this function returning a `bool` if the extension was enabled or not?
(See https://github.com/microsoft/DirectXShaderCompiler/pull/5662/files#diff-44e37c9720575ff94b7842b9ceb70a87fe72486d2b5da2e3828512dc64a352e6R26 for usage)

Reply:
That seems good to me